README.md

```diff
@@ -2,8 +2,9 @@
 
 The toolkit exposes Meta SDKs (e.g Platform SDK), as well as Meta utilities for XR development with the Godot Engine.
 
-**Note:**
-- Use of the toolkit requires **Godot 4.3** or higher.
+> [!WARNING]
+>
+> Use of the toolkit requires **Godot 4.3** or higher.
```

Comment:
Quick note on our discussion about whether this plugin could be used in Godot 4.2.

Reply:
I just tried to build for Godot 4.2, but unfortunately, `EditorExportPlugin::_get_export_options_overrides()` (which we are using) is only in 4.3+

We aren't doing too much with that method at the moment, and so we could consider removing it? However, I'd personally prefer to keep it. If we ever end up with custom export templates, we'll need to make a new version that's Godot 4.3+ anyway, that'll be a very confusing situation.

Reply:
The functionality provided by `_get_export_options_overrides()` is one of the values of this plugin since it simplifies the export preset when the plugin is enabled, so I'd vote for keeping it.

So we can leave the min version at 4.3 then.

Reply:
Sounds good!